app/src/test/java/com/kickstarter/viewmodels/SignupViewModelTest.kt

```diff
@@ -88,25 +125,22 @@ class SignupViewModelTest : KSRobolectricTestCase() {
             ): Observable<AccessTokenEnvelope> {
                 return Observable.error(
                     ApiExceptionFactory.apiError(
-                        builder().httpCode(422).build()
+                        builder().httpCode(422).errorMessages(Collections.singletonList("Unprocessable Content")).build()
```

Comment:
`Collections.singletonList` can be substituted by `listOf()`, in that way we can skip the wildcard import `import java.util.*`. Ktlint usually fails on wildcard imports, so not sure why didn't fail this one (probably because it's a java import but not 100% sure)